build.gradle

```diff
@@ -1,14 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-  ext.kotlin_version = '1.3.21'
+  ext.kotlin_version = '1.4.32'
   repositories {
     google()
     jcenter()
```

Comment:
Is it possible to remove jcenter here?

Reply:
There is one more below, would be great to also remove it
I checked, seems that `com.github.dcendents:android-maven-gradle-plugin` is available on mavencentral. But it's deprecated project anyway, so maybe we can remove it?